src/mod/externals/Dpr/UI/SettingWindow.h

```diff
@@ -26,7 +26,7 @@ namespace Dpr::UI {
             int32_t _selectIndex;
             System::Collections::Generic::List$$SettingMenuItem::Object* _activeItems;
             int32_t _scrollIndex;
-            DPData::CONFIG::Object _tempConfig;
+            DPData::CONFIG::Object* _tempConfig;
```

Comment:
This not being a pointer was correct, since CONFIG is a struct. See the offsets for the fields in dump.cs:
```
private List<SettingMenuItem> _activeItems; // 0x98
private int _scrollIndex; // 0xA0
private CONFIG _tempConfig; // 0xA4
private AudioInstance _voiceInstance; // 0xD8
```